packages/documentation-v7/src/stories/components/collapsible/collapsible.stories.ts

```diff
@@ -0,0 +1,141 @@
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
+import { spread } from '@open-wc/lit-helpers';
+// @ts-ignore
+import { useArgs } from '@storybook/preview-api';

```

Comment:
There is no type at the moment. Documentation is in a work in progress state, see https://github.com/storybookjs/storybook/pull/22590 and https://github.com/storybookjs/storybook/issues/12006